src/wllama.ts

```diff
@@ -76,44 +76,140 @@ export interface WllamaConfig {
 }
 
 export interface WllamaChatMessage {
+  /**
+   * The role of the message
+   */
   role: 'system' | 'user' | 'assistant';
+  /**
+   * The content of the message
+   */
   content: string;
 }
 
 export interface AssetsPathConfig {
+  /**
+   * Path or URL to the single threaded Wllama wasm file
+   */
   'single-thread/wllama.wasm': string;
+  /**
+   * Path or URL to the multi threaded Wllama wasm file
+   */
   'multi-thread/wllama.wasm'?: string;
 }
 
 export interface LoadModelConfig {
+  /**
+   * Seed used by the sampler and other random processes
+   *
+   * @defaultValue 0xFFFFFFFF
+   */
   seed?: number;
+  /**
+   * Context window size
+   *
+   * @defaultValue 1024
+   */
   n_ctx?: number;
+  /**
+   * Prompt processing maximum batch size
+   *
+   * @defaultValue 2048
+   */
   n_batch?: number;
-  // by default, on multi-thread build, we take half number of available threads (hardwareConcurrency / 2)
+  /**
+   * Number of threads to use for generation
+   *
+   * @defaultValue navigator.hardwareConcurrency / 2 on multi-thread build
+   */
   n_threads?: number;
+  /**
+   * Enabled embedding extraction (together with logits)
+   *
+   * @defaultValue false
+   */
   embeddings?: boolean;
+  /**
+   * Offload the KQV ops (including the KV cache) to GPU
+   *
+   * @defaultValue false
```

Comment:
in Llama.cpp, the default value is true. But since Wllama runs only on CPU, i supposed that the default value is `false`